scripts/ci/tags.yaml

```diff
@@ -612,3 +611,49 @@ sample_edge_impulse:
     - nrf/lib/wave_gen/
     - nrf/samples/edge_impulse/
     - zephyr/soc/nordic/
+
+ci_applications_nrf_desktop:
+  files:
+    - bootloader/mcuboot/
+    - modules/crypto/mbedtls/
+    - modules/crypto/oberon-psa-crypto/
+    - modules/crypto/tinycrypt/
```

Comment:
Consider simplifying to `modules/crypto` here (no need to explicitly list these 3 subdirectories)

Reply:
Will do.